app/src/main/res/navigation/main_nav.xml

```diff
@@ -259,7 +259,17 @@
         android:id="@+id/confirmRejectionFragment"
         android:name="io.gnosis.safe.ui.transactions.details.ConfirmRejectionFragment"
         android:label="ConfirmRejectionFragment"
-        tools:layout="@layout/fragment_confirm_rejection" />
+        tools:layout="@layout/fragment_confirm_rejection" >
+
+        <!-- TODO: Add action to go to tx list -->
+        <action
+            android:id="@+id/action_confirmRejectionFragment_to_transactionsFragment"
+            app:destination="@id/transactionsFragment" />
```

Comment:
we need to pop tx details and rejection confirmation fragments from stack. Either directly in code or by adding popUpTo to action

Reply:
Done in  5fc0c7289b0bb3958f5b047015e4d2d3c9ebd93a